api/views.py

```diff
@@ -317,11 +322,12 @@ def instructors_over_time(self, request, format=None):
 
         badges = Badge.objects.instructor_badges()
 
-        qs = Person.objects.filter(badges__in=badges)  \
-                           .annotate(date=Min('award__awarded'),
-                                     count=Value(1,
-                                                 output_field=IntegerField())) \
-                           .order_by('date')
+        qs = Person.objects.filter(badges__in=badges)
+        filter = InstructorsOverTimeFilter(request.GET, queryset=qs)
+        qs = filter.qs.annotate(date=Min('award__awarded'),
+                                count=Value(1,
+                                            output_field=IntegerField())) \
+                      .order_by('date')
```

Comment:
Minor note: this looks strange. I think you can write the same code in a little bit more appealing way:

```python
    filter = InstructorsOverTimeFilter(
        request.GET,
        queryset=Person.objects.filter(badges__in=badges),
    )
    qs = filter.qs.annotate(
        date=Min('award__awarded'),
        count=Value(1, output_field=IntegerField()),
    ).order_by('date')
```